File: src/client/src/app/modules/admin/identity/models/userBranch.ts
export interface UserBranch {
    userBranchs: UserBranchModel[];
}
export interface UserBranchModel {
    identityUserId: string;
    active: boolean;
    selected: boolean;
    organizationId: string;
    branchId: string;
    branchName: string;
}

File: src/server/Modules/Identity/Modules.Identity.Core/Entities/UserBranch.cs
using FluentPOS.Shared.Core.Domain;
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace FluentPOS.Modules.Identity.Core.Entities
{
    public class UserBranch : BaseEntity
    {
        public Guid IdentityUserId { get; set; }

        public bool Active { get; set; }

        [NotMapped]
        public bool Selected { get; set; }
    }
}